src/Telegram.Bot/Requests/Stickers/UploadStickerFileRequest.cs

```diff
@@ -1,6 +1,6 @@
+using System.Net.Http;
```

Comment:
is there a .editorconfig rule for this?

Reply:
There is but I couldn't force Rider to respect it.